pydis_site/apps/api/models/bot/metricity.py

```diff
@@ -89,3 +91,41 @@ def total_message_blocks(self, user_id: str) -> int:
             raise NotFound()
 
         return values[0]
+
+    def top_channel_activity(self, user_id: str) -> List[Tuple[str, int]]:
+        """
+        Query the top three channels in which the user is most active.
+
+        Help channels are grouped under "the help channels",
+        and off-topic channels are grouped under "off-topic".
+        """
+        self.cursor.execute(
+            """
+            SELECT
+                CASE
+                    WHEN channels.name ILIKE 'help-%%' THEN 'the help channels'
+                    WHEN channels.name ILIKE 'ot%%' THEN 'off-topic'
```

Comment:
Grammar here is irking me a bit.
```suggestion
                    WHEN channels.name ILIKE 'help-%%' THEN 'Help channels'
                    WHEN channels.name ILIKE 'ot%%' THEN 'Off topic'
```

Reply:
The purpose of those names is to be used in a sentence, rather than titles, although I agree it's not the prettiest

Reply:
Ah, gotcha, I thought it would be more a sort of list like ```
Help channels: 24,404 messages
Off topic: 12,000 messages
...: ... messages
```